### PR TITLE
Fixed bugs reported during QA

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by kkrampa** [comment]
@emord

**Comment by emord** [comment]
This has a conflict

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -873,7 +869,7 @@ def get(self, request, *args, **kwargs):
 class AWCDailyStatusView(View):
     def get(self, request, *args, **kwargs):
         step = kwargs.get('step')
-        now = datetime.utcnow() - relativedelta(day=1)
+        now = datetime(2017, 6, 28)
```
Comment (emord) [inline custom/icds_reports/views.py]: This should not be hardcoded

**Comment by emord** [comment]
(fixed this in linked pr)

**Pull request closed without merging**